libr/core/cmd_search.c

```diff
@@ -2306,18 +2306,18 @@ static int memcmpdiff(const ut8 *a, const ut8 *b, int len) {
 	return diff;
 }
 
-static void search_similar_pattern_in(RCore *core, int count, ut64 from, ut64 to) {
+static void search_similar_pattern_in(RCore *core, int count, ut64 from, ut64 to, const ut8* block) {
```

Comment:
it doesnt make sense to pass from, to and block without length and the order of arguments is also confusing